src/applications/static-pages/ask-va/components/App/index.js

```diff
@@ -0,0 +1,51 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+// Relative imports.
+import {
+  deriveLOA1URL,
+  deriveLOA2PlusURL,
+  deriveDefaultURL,
+} from '../../helpers';
+import { selectProfile } from 'platform/user/selectors';
+
+export const App = ({ loa, hidden }) => {
```

Comment:
I enjoy the simplicity of the app and utilizing the helpers in an adjacent file.